ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/action-buttons/action-buttons.html

```diff
@@ -44,10 +44,18 @@ <h4 class="ui header" data-sly-test="${!properties['hideLabel']}">${properties['
                     data-asset-share-asset="${asset.path}"
                     data-asset-share-license="${asset.properties['license']}"
                     data-sly-test="${properties['cartLabel'] && config.cartEnabled}">
-                ${properties['cartLabel']}
+                ${properties['addToCartLabel']}
+            </button>
+            <button class="ui button cmp-action-buttons__button "
+		    style="display: none;"
```

Comment:
I think there is a CSS class that will hide; either "hide" or "hidden"?

Reply:
Used hidden class